### DISCUSSION TITLE
Bitmap editor - should we use pointer events?

### DISCUSSION THREAD

**Discussion by BurningTreeC** [TiddlyWiki/TiddlyWiki5#5924] [Ideas]
Currently the bitmap editor uses touch and mouse events to handle the drawing of strokes
Wouldn't it be more appropriate to use pointer events for that?
We could also use pointer pressure to adjust the width of the strokes accordingly

**Comment by pmario** [comment]
Would probably a nice addition

**Comment by Jermolene** [comment]
Hi @BurningTreeC yes, I think that would be useful.

But if we're going to touch the bitmap editor it would be good to make some more extensive fixes. The current design can lose strokes under some situations, I'll have to dig into it again to remember the details.

**Reply by BurningTreeC** [reply]
Ok thanks @Jermolene
